syntax="proto3";

service TestService {
  rpc PredictImage (ImageRequest) returns (ImageResponse);
}

message ImageRequest {
  bytes image_data = 1;
}

message ImageResponse {
  string prediction = 1;
}
